### PR TITLE
Update ci file

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Change license
- Add steps back to the CI

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* master
  * **PR #55** <a href="https://app.graphite.dev/github/pr/ScopeLift/flexible-voting/55" target="_blank">Graphite</a> 
    * **PR #56** <a href="https://app.graphite.dev/github/pr/ScopeLift/flexible-voting/56" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/flexible-voting/56?utm_source=stack-comment).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -50,38 +50,80 @@ jobs:
...
-  #         slither-args: --filter-paths "./lib|./test" --exclude naming-convention
-  #
-  #     - name: Upload SARIF file
-  #       uses: github/codeql-action/upload-sarif@v2
-  #       with:
-  #         sarif_file: ${{ steps.slither.outputs.sarif }}
+        run: |
+          scopelint --version
+          scopelint check
+  coverage:
```
Comment (mds1) [inline .github/workflows/ci.yml:68]:
```suggestion

  coverage:
```

**Comment by mds1** [comment]
> This pull request sets up GitHub code scanning for this repository. Once the scans have completed and the checks have passed, the analysis results for this pull request branch will appear on [this overview](/ScopeLift/flexible-voting/security/code-scanning?query=pr%3A56+is%3Aopen). Once you merge this pull request, the 'Security' tab will show more code scanning analysis results (for example, for the default branch). Depending on your configuration and choice of analysis tool, future pull requests will be annotated with code scanning analysis results. For more information about GitHub code scanning, check out [the documentation](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning). 

Interesting, seems code scanning results no longer show up in-line on PRs. Looking at the linked findings:
- All "high" ones should be addressed by using `safeTransferFrom`
- Numbers 4, 6, 7, 10: can be dismissed 
- Numbers 8, 9: If we aren't using these functions, delete them
- Number 5: We probably should return the number of votes casted (the return value from `GOVERNOR.castVoteWithReasonAndParams`)

If you and @apbendi agree with that assessment, let's use the "Create issue" button from the respective alerts to create issues to track the fixes, and the "Dismiss alert" button with the correct reason otherwise

File: .github/workflows/ci.yml (hunk 1 above)
Comment (alexkeating) [reply .github/workflows/ci.yml:69]: https://github.com/ScopeLift/flexible-voting/pull/56/commits/153c37be678edf8567f8e4c7565b6d1694855ddc

**Review by apbendi — APPROVED** [review]
Looks good to me!

**Pull request merged**